README.md

```diff
@@ -1,65 +1,283 @@
+| MinIO       | minio                  | minio-tenant-1   | ClusterIP    | 443             |
+| Console     | minio-tenant-1-console | minio-tenant-1   | ClusterIP    | 9090,9443       |
++-------------+------------------------+------------------+--------------+-----------------+
+
 ```
 
-Optionally, you can generate a yaml file with the `-o` flag in above command and modify the yaml file as per your specific requirements. Once you verify and optionally add any other relevant fields to the file, create the tenant
+Copy the credentials to a secure location, such as a password protected key manager. 
+MinIO does *not* display these credentials again.
+
+MinIO Tenants deploy with TLS enabled by default, where the MinIO Operator uses the 
+Kubernetes `certificates.k8s.io` API to generate the required x.509 certificates. Each 
+certificate is signed using the Kubernetes Certificate Authority (CA) configured during
+cluster deployment. While Kubernetes mounts this CA on Pods in the cluster, Pods do 
+*not* trust that CA by default. You must copy the CA to a directory such that the 
+`update-ca-certificates` utility can find and add it to the system trust store to 
+enable validation of MinIO TLS certificates:
 
 ```sh
-kubectl minio tenant create --name tenant1 --servers 4 --volumes 16 --capacity 16Ti -o > tenant.yaml
-kubectl apply -f tenant.yaml
+
+cp /var/run/secrets/kubernetes.io/serviceaccount/ca.crt /usr/local/share/ca-certificates/
+update-ca-certificates
 ```
 
-## Post Tenant Creation
+For applications *external* to the Kubernetes cluster, you must configure 
+[Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) or a
+[Load Balancer](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) to
+expose the MinIO Tenant services. The `kubectl port-forward` command temporarily forwards traffic
+from the local host to the MinIO Tenant. 
+
+- The `minio` service provides access to MinIO Object Storage operations.
+
+- The `minio-tenant-1-console` service provides access to the MinIO Console. The 
+  MinIO Console supports administration of the MinIO Tenant.
+
+# Expand a MinIO Tenant
+
+MinIO supports expanding an existing MinIO Tenant onto additional hosts and storage.
+
+- MinIO requires Kubernetes version 1.17.0 or later.
+
+- This procedure assumes the cluster contains a 
+  [namespace](https://github.com/minio/operator#create-a-namespace) for
+  the MinIO Tenant.
+
+- This procedure assumes the cluster contains a
+  [`StorageClass`](https://github.com/minio/operator#default-storage-class)
+  for the MinIO Tenant Persistent Volume Claims (`PVC`).
 
-### Expanding a Tenant
+The following `kubectl minio` command expands a MinIO Tenant with an additional
+4 nodes, 16 volumes, and added capacity of 16Ti. This configuration requires
+*at least* 16 [Persistent Volumes](https://github.com/minio/operator#Local-Persistent-Volumes).
```

Comment:
Should this say 16 new?  Also, it is possible for readers to misconstrue the requirement of 16 as a hard limit in minio rather than just what is required for this example

Reply:
Made some tweaks to hopefully make it clear this is driven by the number of PVC